src/vaadin-upload.html

```diff
@@ -809,7 +809,7 @@
         * @event file-reject
         * @param {Object} detail
         * @param {Object} detail.file the file added
-        * @param {Object} detail.error the cause
+        * @param {String} detail.error the cause
```

Comment:
`string` with a small `s` as per the [Jsdoc spec](https://devhints.io/jsdoc).

Reply:
I see, thanks! Done :)